### PR TITLE
Feature BL-4454 start page content update

### PR DESCRIPTION
**Pull request by paddyfox**
Updated content for the MOT reminders start page
Modified Selenium test to reflect new content changes

### CODE REVIEW

File: webapp/src/main/resources/template/home.hbs [hunk 1]
```diff
@@ -3,31 +3,48 @@
...
+                <p>You’ll get a reminder a month before your MOT expiry date. You can take the test up to a month
+                (minus a day) before it expires.</p>
+                <p>Without a valid MOT you can’t tax your vehicle and you could get a fine.</p>
+            </div>
+        <form action="#">
+            <nav class="content-navigation">
+                <a href="/mot-reminders/car-details" id="startButton" class="button">Start now</a>
+                <ul class="content-navigation__secondary">
+                </ul>
+            </nav>
```
Comment (Thomas-Franklin) [inline webapp/src/main/resources/template/home.hbs]: This secondary ul class is un-needed as we won't have a back button on this page - remove it maybe?

Comment (paddyfox) [inline webapp/src/main/resources/template/home.hbs]: Agreed. Good point, I will remove this 

Thanks

Comment (aaronlong) [inline webapp/src/main/resources/template/home.hbs]: Might be worth checking with one of the front-end people to see if this is needed or was included for a reason :)

**Comment by przemos** [comment]
Squash your commits please.

**Comment by Thomas-Franklin** [comment]
@przemos there is a squash and merge option in github - does this do an automatic squash for us? Or will it be best for us to manually do it?

**Pull request merged**